cephfs/admin/volume_test.go

```diff
@@ -12,19 +12,25 @@ func TestListVolumes(t *testing.T) {
 
 	vl, err := fsa.ListVolumes()
 	assert.NoError(t, err)
-	assert.Len(t, vl, 1)
-	assert.Equal(t, "cephfs", vl[0])
+	assert.GreaterOrEqual(t, len(vl), 1)
+	assert.Contains(t, vl, "cephfs")
```

Comment:
Just a nit: I bet `Contains` implies a `len()` check. Before it was necessary, because the `Equal()` directly dereferenced `vl[0]`.

Reply:
Reasonable. But I think it does no harm. I'll change it only if I need to touch the file for another reason.